### DISCUSSION TITLE
SSG: Redirect to defined pages base on form submit

### DISCUSSION THREAD

**Discussion by kr8n3r** [vercel/next.js#14224] [Help] [answered]
users can select one of two locations.
Looking to avoid javascript for this. Any built-in ways or would I have to use say express?

**Comment by enricoschaaf** [comment]
No need to use Express, Node can do this all by itself, using the writeHead method. `res.writeHead(301,
  {Location: "www.example.com"}
).end()`
https://nodejs.org/api/http.html#http_response_writehead_statuscode_statusmessage_headers

**Reply by kr8n3r** [reply]
redirect:yes, but passing form data to base the redirection on: no

**Reply by enricoschaaf** [reply]
Can you elaborate what you mean by passing form data? In my head a user submits a form, you get that info on the server and based on the data you can redirect the user, what am I missing?

**Reply by kr8n3r** [reply]
ended up using form `post` without `action`. that way I get url with post params like `?param=value` and use ngix to redirect based on those params

**Comment by kr8n3r** [comment, marked as answer]
ended up using form `post` without `action`. that way I get url with post params like `?param=value` and use ngix to redirect based on those params
